spec/rails_helper.rb

```diff
@@ -27,7 +27,7 @@
   include FactoryBot::Syntax::Methods
 
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.fixture_path = Rails.root.join('/spec/fixtures')
```

Comment:
Do we need a forward slash before the segment "spec" or not?

Reply:
I think not, I'll remove it to leave it relative